packages/fxa-auth-server/lib/senders/emails/partials/button/index.mjml

```diff
@@ -5,7 +5,7 @@
 <mj-include path="./css/button/index.css" type="css" css-inline="inline" />
 
 <mj-section>
-  <mj-column>
+  <mj-column css-class="<%= locals.cssClass || undefined %>">
```

Comment:
Only because this looks ever so slightly funky, my first inclination was to recommend...

```
<% if (locals.cssClass) { %> <%= locals.cssClass %> <% } %>
```

or (less confident about this one without testing), `<% locals.cssClass && locals.cssClass %>`?

Though honestly what you have is probably fine so long as `undefined` isn't actually output, and I don't think it is. In React we oftentimes set default classes to `undefined` so `class` isn't rendered on the DOM element so yeah, maybe if my second example works it could be slightly preferable but eh 🤷‍♀️

Reply:
I checked that `undefined` is not actually in the output. 🙂